generators/app/index.js

```diff
@@ -95,6 +95,23 @@ module.exports = generators.Base.extend({
       }.bind(this));
     },
 
+    askForGithubAccount: function () {
+      var done = this.async();
+
+      githubUsername(this.user.git.email(), function (err, username) {
+        this.prompt({
+          name: 'githubAccount',
+          message: 'GitHub username or organization',
+          default: username
+        }, function (prompt) {
+          if (prompt.githubAccount) {
```

Comment:
If githubAccount is a falsy value, our generator will hang forever in the current setup.